### DISCUSSION TITLE
Automating NPC/Host Timeline Generation via Animations

### DISCUSSION THREAD

**Discussion by dh4340** [cmu-sei/GHOSTS#432] [General]
Hey,

I've been exploring the timeline generation feature for NPCs and hosts within our project. I'm interested in automating this process, specifically by pushing timelines for each NPC/Host through the Animations autonomously.

Is it possible to implement a feature that can automatically generate and push these timelines? If so, I would love to discuss the feasibility of this idea and any potential steps we could take to create it.

Looking forward to your thoughts!

Thank you!

**Comment by sei-dupdyke** [comment]
Hi, can you provide more detail? What kind of automation do you mean? What kinds of timelines do you want to generate?

**Reply by dh4340** [reply]
So for example, create a normal user activity for an NPC. Which is auto generated? So timelines don't have to be created one by one. And aren't templated.

**Comment by sei-dupdyke** [comment]
So some sort of timeline distributor that provides some amount of configuration (this kind of activity on some frequency to all or groups of machines) and then puts those timelines out for delivery?

Seems reasonable.

**Comment by sei-dupdyke** [comment]
Issue #434 created.
